crates/context/src/journaled_state/init.rs

```diff
@@ -1,49 +0,0 @@
-use super::Journal;
-use database_interface::EmptyDB;
-
-/// A cloneable version of JournaledState that uses EmptyDB.
-/// Used to clone the journaled state and for initialization of new journaled state.
-pub type JournalInit = Journal<EmptyDB>;
```

Comment:
The removal of this type may be unintended, or is it not the correct way to initialize with an EmptyDB anymore

Reply:
There is no need for it as `JournalInner` now contains all journaled statea and has same purpose as `JournalInit`